presto-docs/src/main/sphinx/release/release-0.256.rst

```diff
@@ -0,0 +1,29 @@
+=============
+Release 0.256
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add support for lambda expressions in ``PREPARE`` statements.
+* Add configuration property ``experimental.distinct-aggregation-spill-enabled`` and session property ``distinct_aggregation_spill_enabled`` to disable spilling for distinct aggregations.
+* Add configuration property ``experimental.order-by-aggregation-spill-enabled`` and session property ``order_by_aggregation_spill_enabled`` to disable spilling for order by aggregations.
+* Add double quotes around the column domain values in the text query plan. Literal double quotes in values will be escaped with a backslash (``ab"c`` -> ``ab\"c``).
+* Add function :func:`array_frequency`.
+* Add support for :func:`zip` to take upto 7 arguments.
+* Add support for ``REFRESH MATERIALIZED VIEW``.
+
+Web UI Changes
+______________
+* Add ``Cumulative Total Memory`` to the ``Resource Utilization Summary`` table on the query detail page.
```

Comment:
should be "Query Details" page (plural).  maybe in double back tick also since it's the page heading

Reply:
fixed